CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```
+$ git clone git@github.com:<username>/cryostatio.github.io.git
+$ git remote add upstream git@github.com:cryostatio/cryostatio.github.io.git # Adding upstream as remote
+```
+Checkout another branch.
+```
+$ cd cryostatio.github.io && git checkout -b some-task-branch
+```
+Edit some files and commits your changes.
+```
+$ git add . # Stage your changes. Here, we stage the entire repo (recursively).
+$ git commit -m "some very nice message"
+```
+
+## Open a PR
+
+Push your changes to your remote respository (forked). You might need to rebase on the newest commit on the upstream.
+
+To rebase:
+```
+$ git fetch upstream # Fetch upstream commits
+$ git checkout main && git merge upstream/main && git push origin main # Checkout main, merge upstream and update origin
```

Comment:
`merge` and `rebase` are different, best to have the command and the description match up. For the purpose of opening a PR either is fine, though personally I prefer to rebase.

Reply:
Right! Fixed that!